test/pages/PersonalProfile.test.jsx

```diff
@@ -5,6 +5,8 @@ import { expect, it, describe } from 'vitest';
 import { CivicProfile } from '@pages';
 import { CIVIC_FORM_LIST } from '@components/CivicProfileForms';
 
+// TODO: Shouldn't this be testing Profile.jsx rather than CivicProfile.jsx?
```

Comment:
Also if there's a specific test file for CivicProfile, think we could drop the test here